deps/jerry/api.cc

```diff
@@ -1965,8 +1965,19 @@ MaybeLocal<Value> v8::Object::GetOwnPropertyDescriptor(Local<Context> context,
   V8_CALL_TRACE();
   JerryValue* jobject = reinterpret_cast<JerryValue*>(this);
   JerryValue* jkey = reinterpret_cast<JerryValue*>(*key);
+  JerryValue* property_object;
 
-  JerryValue* property_object = jobject->GetOwnPropertyDescriptor(*jkey);
+  jerry_property_descriptor_t prop_desc;
```

Comment:
prop_desc should be initialized with  `jerry_init_property_descriptor_fields (&prop_desc);`